mycore-acl/src/main/java/org/mycore/mcr/acl/accesskey/MCRAccessKeyManager.java

```diff
@@ -103,6 +103,9 @@ public static boolean isValidSecret(final String secret) {
      * @throws MCRException if encryption fails
      */
     public static String hashSecret(final String secret, final MCRObjectID objectId) throws MCRException {
+        if (HASH_ITERATIONS == 0) {
```

Comment:
maybe `-1` would also make sense for the deactivation, because iteration actually means to repeat or a separate new property for it.